### PR TITLE
Fix contributions batch actions

### PR DESCRIPTION
**Pull request by andreLumor**
Related issue: #123 

This PR fixes the contributions batch approve and reject.

Observed Problem:
Every review resulted in three database calls, one on approve or reject action, and then two called by SaveContributionReviewer service object.

Proposed changes:
Remove SaveContributionReviewer service object.
Contribution's approve and reject now receive the reviewer's id as argument.
Create a proc on contribution approve and reject that updates reviewer info.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ gem 'rails', '~> 6.1.5.1'
 
 gem 'bootsnap', require: false
 
+gem 'rack', '~> 2.2.3.1'
```
Comment (adbatista) [inline Gemfile:13]: you don't need this

Comment (andreLumor) [reply Gemfile:13]: This update was due to CI security

Comment (adbatista) [reply Gemfile]: is a dependency of rails, if you just do `bundle update rack` will update the gemfile.lock and you don't need to add the gem here

Comment (andreLumor) [reply Gemfile]: Oh, good to know!

It is already fixed.

**Pull request merged**